### PR TITLE
Edit UG and DG

### PR DESCRIPTION
**Pull request by tsumian**
Tidy up UG with updated SS and command table.

Update DG. Edit filter command implementation with class diagram.
remove irrelevant user stories. Add more use cases. Add more test
cases for manual testing section. Add effort section in appendix.

### CODE REVIEW

File: docs/DeveloperGuide.md [hunk 1]
```diff
@@ -639,36 +769,211 @@ testers are expected to do more *exploratory* testing.</div>
...
+After many rounds of refinement, the current `Model` of WoofAreYou is one of the more efficient and scalable model
+catered for its purpose.
+
+## Logic
+
+WoofAreYou's `Logic` is more advanced than that of AB3. WoofAreYou supports many more commands such as `sort`,
+`filter`, `undo`, `diet`, `app`, and the different commands for marking attendance. Existing commands of AB3 like
+`edit` and `find` were also enhanced and adapted to handle the complex `Model` of WoofAreYou.
+
+Significant effort was placed in implementing the `Logic` for WoofAreYou. It encompasses the logic for pets'
```
Comment (tobihy) [inline docs/DeveloperGuide.md:911]: Past tense here...

File: docs/DeveloperGuide.md [hunk 2]
```diff
@@ -639,36 +769,211 @@ testers are expected to do more *exploratory* testing.</div>
...
+have. `AttendanceEntry`, `Appointment`, `Diet` and `Tags` were carefully organised and stored as separate entries in a
+JSON file. This ensures that these attributes are unique to each pet and can be retrieved easily for other implementations.
+
+More time and effort were spent in crating the JSON entries of `AttendanceEntry` as it needed to store dates that
+the pet was present and absent, and also the pick-up and drop off timings of different pets. All these needed to be
+encapsulated in an organised format in the JSON file for easy retrieval.
+
+## UI
+
+WoofAreYou's `UI` is designed to be very dependent on visual cues to enhance user-friendliness. Every command that the
```
Comment (tobihy) [inline docs/DeveloperGuide.md:931]: ...But present tense here!

**Review by tobihy — COMMENTED** [review]
Some minor nits found. 
One issue I noted was the inconsistency in verb tenses for the "Effort" section! 
Other than that, LGTM! Good work.

File: docs/DeveloperGuide.md [hunk 3]
```diff
@@ -639,36 +769,211 @@ testers are expected to do more *exploratory* testing.</div>
...
+user execute will be accompanied by a status message at the very least. This status message appears at the top of
+WoofAreYou to help users understand what is going on after each execution. In the event of any invalid input, error
+messages shown are crafted to be intuitive and easily understandable so that users can rectify their mistakes.
+
+### Attendance Visual Cues
+
+WoofAreYou has a dedicated column on the right to indicate a pet's attendance. Significant amount of effort is put into
```
[the remark this answers is not in the document]
Comment (tsumian) [reply docs/DeveloperGuide.md:938]: Will read through it again for consistency

**Pull request closed without merging**